packages/now-build-utils/src/detect-builders.ts

```diff
@@ -210,5 +211,6 @@ export async function detectBuilders(
   return {
     builders: builders.length ? builders : null,
     errors: errors.length ? errors : null,
+    warnings,
```

Comment:
Why are builders and errors nullable but warnings is never null?

Reply:
Since they can always be there, builders exclude errors and errors exclude builders.